pkg/tfvars/internal/cache/cache.go

```diff
@@ -12,16 +12,19 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/h2non/filetype"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	"github.com/ulikunitz/xz"
 
 	"golang.org/x/sys/unix"
 )
 
 const (
 	applicationName = "openshift-installer"
 	imageDataType   = "image"
-	gzipFileType    = "application/x-gzip"
+	gzipFileType    = "application/gzip"
```

Comment:
is it possible to use the consts like https://github.com/h2non/filetype/blob/8241aa2c448465326c0002ccc6df7b910658960d/matchers/archive.go#L8 ?